src/lang/bg-BG.json

```diff
@@ -174,20 +174,20 @@
   "game_mode_6":"Intro",
   "game_mode_7":"Diretide",
   "game_mode_8":"Reverse Captains Mode",
-  "game_mode_9":"Greeviling",
-  "game_mode_10":"Tutorial",
+  "game_mode_9":"The Greeviling",
```

Comment:
You might want to backport these string updates to the English file, otherwise English will not be consistent with the translated files.